accountservice/src/main/java/io/marius/demo/ecommerce/accountservice/service/AuthorizationService.java

```diff
@@ -1,7 +1,8 @@
 package io.marius.demo.ecommerce.accountservice.service;
 
 import io.marius.demo.ecommerce.accountservice.model.payload.LoginPayload;
+import org.springframework.http.ResponseEntity;
 
```

Comment:
you don't need this interface unless you want to perform inheritance with more than 1 implementation